internal/node/node_repositories.bzl

```diff
@@ -176,44 +178,57 @@ exports_files([
 alias(name = "node", actual = "{node}")
 alias(name = "npm", actual = "{npm}")
 alias(name = "yarn", actual = "{yarn}")
+filegroup(
+  name = "node_runfiles",
+  srcs = glob(
+    [
+      "bin/node.js",
+      "{node_dir}/**",
+      "{yarn_dir}/**",
+    ],
+    exclude = [
```

Comment:
I assume you checked if there was anything more we could exclude, to try to bring down the count/size of the action input files